ios/Capacitor/Capacitor/CAPFile.swift

```diff
@@ -11,7 +11,7 @@ public class CAPFile {
  * disk, in a bundle, or in another location.
  */
 @objc public class CAPFileManager: NSObject {
-    static func get(path: String) -> CAPFile? {
+    public static func get(path: String) -> CAPFile? {
```

Comment:
Can this just return a `URL`? CAPFile is a pretty useless wrapper since it only has one property. In addition, using that object means that this method can't be exported to Obj-C because `CAPFile` is Swift-only.